components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -108,8 +126,11 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	}, reconcileErr
 }
 
-func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alpha1.Subscription) (bool, error) {
+func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alpha1.Subscription,
+	KymaSubscriptionsGauge *metrics.SubscriptionsGauge) (bool, error) {
 
+	KnativeSubscriptionsGauge := metrics.KnativeSubscriptionsGaugeObj
```

Comment:
Why does this local variable start with uppercase?

Reply:
will fix